lib/model/card/action/scry_action.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
```

Comment:
Can use relative import here (shorter and consistent with the other files)

Reply:
Makes sense, I can go through and tidy all of these up. Might do it as a separate commit though.